.github/workflows/release.yml

```diff
@@ -16,6 +16,7 @@ jobs:
         with:
           node-version: 12.x
           registry-url: 'https://registry.npmjs.org'
+          cache: npm
```

Comment:
I guess it's not necessary here since we're not running `npm install`?

Reply:
```suggestion
```